docker-compose.yaml

```diff
@@ -90,6 +90,18 @@ services:
       DD_API_KEY: invalid_key_but_this_is_fine
     ports:
       - "8126:8126"
+  testagent:
+    image: williamconti549/dd-apm-test-agent:service-naming
```

Comment:
Can we use an official datadog registry? For better security and maintenance.

Reply:
will update to use final test agent image